packages/core/src/2d/sprite/SpriteMask.ts

```diff
@@ -238,8 +238,9 @@ export class SpriteMask extends Renderer {
   }
 
   protected override _updateBounds(worldBounds: BoundingBox): void {
-    if (this.sprite) {
-      SimpleSpriteAssembler.updatePositions(this);
+    const { sprite } = this;
+    if (sprite) {
+      SimpleSpriteAssembler.updatePositions(this, this.width, this.height, sprite.pivot, this._flipX, this._flipY);
```

Comment:
Destructuring assignment usually takes out the attributes to be reused to avoid repeated reading, which does not exist here.

The code itself is not very complex and can be written in one line, so I don't think it is necessary